src/components/action/README.md

```diff
@@ -0,0 +1,48 @@
+# Action
+
+Provide visible and concise path forward to a user
```

Comment:
Should this be "Provide **a** visible..."? It reads a bit awkward to me the way it is.

Reply:
Yup, that's totally a typo, lol.